README.md

```diff
@@ -22,8 +25,11 @@ console.log(results);
 ```
 
 ```js
-// Your answer...
+var results = AuthorModel.findOne({name: "Bob"});
+console.log(results);
+
 ```
+.find method will result in an array.
```

Comment:
`findOne` requires a callback as a second argument in order for `console.log` to actually print out `results`. Asynchronicity!